server/restapi/restapi.coffee

```diff
@@ -0,0 +1,59 @@
+Api = new Restivus
+	useDefaultAuth: true
+	prettyJson: true
+
+
+Api.addCollection ChatMessage
+Api.addCollection ChatRoom
+
+
+Api.addRoute 'version', authRequired: false,
+	get: ->
+		version = {api: '0.1', rocketchat: '0.5'}
+		status: 'success', versions: version
+
+Api.addRoute 'publicRooms', authRequired: true,
+	get: ->
+		rooms = ChatRoom.find({ t: 'c' }, { sort: { msgs:-1 } }).fetch()
+		if rooms
```

Comment:
I don't know, but maybe we can add new params to allow caller to pass start and limit.
Or are you talking about limit the fields?